src/storage/v3/indices.hpp

```diff
@@ -114,8 +122,35 @@ class LabelIndex {
 
   void Clear() { index_.clear(); }
 
+  std::map<IndexType, IndexContainer> SplitIndexEntries(const PrimaryKey &split_key) {
+    if (index_.empty()) {
+      return {};
+    }
```

Comment:
Is this check actually needed?
If `index_` is empty, the loop will not be entered and the empty `cloned_indices` will be returned.

Reply:
The same stands for `LabelPropertyIndex::SplitIndexEntries`